docs/documentation/siddhi-4.0.md

```diff
@@ -2345,4 +2352,167 @@ You can use Store extension type to work with data/events stored in various data
                           
           eg:- Define value for property 'storeType': RDBMS
     
-* Finally confirm all property values are correct or not by typing Y or press Enter, else type N
\ No newline at end of file
+* Finally confirm all property values are correct or not by typing Y or press Enter, else type N.
+
+## Configuring and Monitoring Siddhi Applications
+
+This section explains how to use the `@app` annotation to generate statistics for Siddhi applications as well as improve the performance of Siddhi applications.
+
+### @app:statistics
+
+To evaluate the performance of an application, you can enable the statistics of a Siddhi application to be published. This is done via the `@app:statistics` annotation that can be added to a Siddhi application as shown in the following example.
+
+```sql
+@app:statistics(reporter = 'console')
+```
+The following elements are configured with this annotation.
+
+|Annotation| Description| Default Value|
+| ------------- |-------------|-------------|
+|`reporter`|The interface in which statistics for the Siddhi application are published. Possible values are as follows:<br/> `console`<br/> `jmx`<br/> `jdbc`<br/> |`console`|
+|`interval`|The time interval (in seconds) at  which the statistics for the Siddhi application are reported.|`60`|
+|`include`| This allows you to specify the types of metrics to be included in the reporting as a comma-separated list. It is also possible to use wild cards.| All (*.*)|
```

Comment:
Can we add an only here? If include is given, only the metrics matching that pattern will be reported.